pkg/virt-controller/watch/vm.go

```diff
@@ -318,7 +318,7 @@ func (c *VMController) execute(key string) error {
 
 	var syncErr syncError
 
-	syncErr, err = c.sync(vm, vmi, key, dataVolumes)
+	vm, syncErr, err = c.sync(vm, vmi, key, dataVolumes)
```

Comment:
Does this mean we have computed the status below on "old" state previously?

Reply:
Generally no, because in the `sync` method we do not update the vm, but in some cases yes https://github.com/kubevirt/kubevirt/blob/eeb7ea2ab68a8bb44eed0f24a1ced0ba1d4a411b/pkg/virt-controller/watch/vm.go#L2295